src/App.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import TimeStamp from './components/TimeStamp';
+import ChatEntry from './components/ChatEntry';
```

Comment:
Neither of these components need to be imported. You should only import components you're directly using. While `Chatlog` uses `ChatEntry` and `ChatEntry` uses `TimeStamp`, they only need to be imported in the files in which the actual component is being used.